test/python/visualization/references/pass_manager_style.dot

```diff
@@ -13,75 +13,81 @@ fontname=helvetica;
 label="[1] condition";
 labeljust=l;
 4 [color=red, fontname=helvetica, label=TrivialLayout, shape=rectangle];
-5 [color=black, fontname=helvetica, fontsize=10, label=coupling_map, shape=ellipse, style=solid];
+5 [color=black, fontname=helvetica, fontsize=10, label=coupling_map, shape=ellipse, style=dashed];
```

Comment:
Are the changes in the two `dot` files related specifically to this PR?

Reply:
Yeah, because the default pass manager is changing the pass manager drawer test failed, so I just regenerated the expected file from the test.